### PR TITLE
change to readme

### PR DESCRIPTION
**Pull request by mandycandyqt**
ok

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 For Alessio
```
Comment (fraczles) [inline README.md]: Uppercase

Comment (mandycandyqt) [inline README.md:1]: i think it's actually ok @fraczles

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
...
+love mandy and alex
```
Comment (fraczles) [inline README.md:2]: Uppercase?

File: README.md (hunk 1 above)
Comment (fraczles) [inline README.md:1]: Oh I think you're right

Comment (mandycandyqt) [inline README.md:1]: yeah i know @fraczles

**Pull request merged**